src/avram/primary_key_queryable.cr

```diff
@@ -0,0 +1,25 @@
+require "./errors"
+
+module Avram::PrimaryKeyQueryable(T)
+  abstract def id
+  abstract def id(id_val)
+  abstract def query
+
```

Comment:
I think these should be removed since the module itself implements the methods. 
```suggestion
```

Reply:
@paulcsmith done